.github/workflows/ci.yaml

```diff
@@ -20,6 +20,10 @@ jobs:
         uses: cachix/install-nix-action@018abf956a0a15673dae4932ae26f0f071ac0944
         with:
           nix_path: nixpkgs=channel:nixpkgs-unstable
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: "v0.4.1"
```

Comment:
unless there's some specific version dep here, use the generic version

Reply:
Fixed.